src/components/createAppointment/createAppointment.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { MainWrapper, Text, Button } from '../mainDoctor/styles'
+import { nanoid } from '@reduxjs/toolkit'
+import Title from '../title/title'
+import { Link, TextWrapper, Wrapper } from './createAppointment.styles'
+import DatePicker from './datePicker/datePicker'
+import TimePicker from './timePicker/timePicker'
+import DoctorPicker from './doctorPicker/doctorPicker'
+
+import userActions from '../../redux/user/userActions'
+
+const CreateAppointment = ({ toggleCreateAppointment }) => {
+  const [dateData, setDateData] = useState()
+  const [timeData, setTimeData] = useState()
+  const [formData, setFormData] = useState()
+  const [disabled, setDisabled] = useState(true)
+
+  const dispatch = useDispatch()
+
+  const handleClick = () => {
+    toggleCreateAppointment(false)
+  }
+
+  const onChangeData = (data, name) => {
+    switch (name) {
+      case 'formData':
+        return setFormData(data)
+
+      case 'timeData':
+        return setTimeData(data)
+
+      case 'dateData':
+        return setDateData(data)
+
+      default:
+        return
+    }
+  }
+
+  const onSubmitAppointment = () => {
+    const data = {
+      id: nanoid(),
+      name: formData.name,
+      status: formData.status,
+      date: `${dateData} ${timeData}`,
+      description: formData.description,
+      note: formData.note,
+    }
+
+    dispatch(userActions.setDoctorsList(data))
+    handleClick()
+  }
+
+  useEffect(() => {
+    if (dateData && timeData && formData?.name && formData?.status) {
+      setDisabled(false)
+    }
+    return
```

Comment:
can't get it what this return for

Reply:
I thought it was good practice if I'm return something ;D